embassy-stm32/src/gpio.rs

```diff
@@ -115,6 +149,8 @@ impl<'d, T: Pin> Drop for Output<'d, T> {
             let n = self.pin.pin() as usize;
             r.pupdr().modify(|w| w.set_pupdr(n, vals::Pupdr::FLOATING));
             r.moder().modify(|w| w.set_moder(n, vals::Moder::INPUT));
+            r.otyper().modify(|w| w.set_ot(n, vals::Ot::PUSHPULL));
```

Comment:
if pin is set as floating input, aren't otyper/ospeedr ignored?

Reply:
Probably, I was just setting again to leave at a reset state for the "next driver".